evm/hardhat.config.ts

```diff
@@ -1,8 +1,70 @@
+import "@nomiclabs/hardhat-ethers";
+import "@nomiclabs/hardhat-etherscan";
 import { HardhatUserConfig } from "hardhat/config";
 import "@nomicfoundation/hardhat-toolbox";
+import dotenv from "dotenv";
+
+dotenv.config();
 
 const config: HardhatUserConfig = {
   solidity: "0.8.9",
+  networks: {
```

Comment:
There's another file in the project with network information like this and I wonder if we want to somehow try to combine them. Perhaps not for this project, but something to consider in enterprise.

Reply:
Maybe we should build our own npm/yarn packages with network/RPC information for our projects, and just include them in each project.